server/src/main/webapp/WEB-INF/rails/config/routes.rb

```diff
@@ -46,31 +46,11 @@
   put "admin/config_xml" => "admin/configuration#update", as: :config_update
   get "admin/config_xml/edit" => "admin/configuration#edit", as: :config_edit
 
-  resources :analytics, only: [:index], controller: "analytics"
-  get 'analytics/:plugin_id/:type/:id' => 'analytics#show', constraints: {plugin_id: PLUGIN_ID_FORMAT, id: PIPELINE_NAME_FORMAT}, as: :show_analytics
```

Comment:
IIRC, this is still used on VSM page. Please check.

Reply:
I dint see the analytics controller, hence I removed it. Will check.